packages/langium/test/parser/langium-parser-builder.test.ts

```diff
@@ -437,25 +437,25 @@ describe('Parser calls value converter', () => {
 
     test('Should parse bool correctly', () => {
         expectValue('b true', true);
-        // this is the current 'boolean' behavior when a prop type can't be resolved to just a boolean
-        // either true/undefined, no false in this case
-        expectValue('b false', undefined);
-        // ...then no distinguishing between the bad parse case when the type is unclear
-        expectValue('b asdfg', undefined);
+        expectValue('b false', false);
+        // Any value that cannot be parsed correctly is automatically false
+        expectValue('b asdfg', false);
```

Comment:
This feels like a breaking change, especially if there's any upstream code that strictly relies on `undefined` to be produced.

Reply:
That doesn't make a lot of sense, since a type that contains `boolean` is guaranteed to be `false` as a fallback (even if it's not parsed). Langium never generates a property as `value?: boolean`. This is actually a bug fix rather than a new feature.

Reply:
I'm aware it was a bug from before, but it's possible it will break something. I should clarify, in the next release notes we should make this change clear; but beyond that the basis for booleans not having an undefined value -- where it should be false -- is clear.